src/components/organisms/MediaBox.jsx

```diff
@@ -11,11 +13,22 @@ const Fullscreen = styled.div`
   width: 100vw;
 `
 
-export const MediaBox = ({ media }) =>
-  !!media?.url > ''
-    ? (media.fullscreen && (
+export const MediaBox = ({ media }) => {
+  return !!media?.url > '' ? (
+    media.url.indexOf('spotify') !== -1 ? (
+      media.fullscreen ? (
         <Fullscreen>
-          <ReactPlayer url={media.url} playing={false} width="100%" height="100%" />
+          <SpotifyPlayer url={media.url} />
         </Fullscreen>
-      )) || <ReactPlayer url={media.url} playing={false} width="100%" height="100%" />
-    : null
+      ) : (
+        <SpotifyPlayer url={media.url} />
+      )
+    ) : media.fullscreen ? (
```

Comment:
not a good programming pattern with ? : ... ? : ... Better would be to use _&&_ and _||_ and I think here even clearer would be to first identify the player (spotify or other) and the return it either fullscreen or not. Plan and simple if else logic ;)

```
const Player = media.url.indexOf('spotify') !== -1 ? SpotifyPlayer : ReactPlayer

return media.fullscreen ? <Fullscreen><Player ... /></Fullscreen> : <Player ... />
```